src/components/filter.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import Form from "react-bootstrap/Form";
+
+class Filter extends React.Component {
```

Comment:
Great work on this so far! 

Feedback here/Food for thought: I would maybe considering renaming this class as it holds the Form logic from bootstrap